### DISCUSSION TITLE
Deleting data and also how can I access Multiple index

### DISCUSSION THREAD

**Discussion by projx** [zinclabs/zinc#424] [Q&A]
Hi All

Does Zinc search support the same methods of index names as elastic, i.e. it can search across:

- myIndex-2022-07-21
- myIndex-2022-07-22
- myIndex-2022-07-23

In elastic, to trim data I would just delete that days index, how should I approach deleting data with ZincSearch?  Currently I use 1 single index (hence why I'm asking about the dates above) and I only want to keep 7 days worth of data, so I'm trying to find a way to trim it down.

Sorry if this is a silly question, its been a while since I touched ElasticSearch, so I'm rusty, and I'm looking to use ZincSearch in a small personal project.

**Comment by prabhatsharma** [comment]
You can delete the index using the GUI or by using the [API](https://docs.zincsearch.com/api/index/delete/)
